opcodes/arc-dis.c

```diff
@@ -117,6 +117,11 @@ typedef struct skipclass
    disassembling.  */
 static linkclass decodelist = NULL;
 
+/* ISA mask value enforced via disassembler info options.  ARC_OPCODE_NONE
+   value means that no CPU is enforced.  */
+
+static unsigned enforced_isa_mask = ARC_OPCODE_NONE;
```

Comment:
we can configure binutils project with a specific CPU (--with-cpu=<name>) is not better to have the configured CPU here?